### DISCUSSION TITLE
Unified box

### DISCUSSION THREAD

**Discussion by serrq** [tutao/tutanota#7490] [Ideas]
When I set "inbox" rules I get emails separation. Each incoming email goes in the proper drawer (category).

This is good.

But I need also of an unified box to see my emails just all together, in chronological way.

Currently I haven't a way to perform this, if I set rules.

I want both ways, simultaneously.

**Comment by ganthern** [comment]
Hey there :wave"

We're currently re-architecting the way we store e-mails and this "All Mail" view is one thing that's made easier by these changes. Most of it is already live, but there's no ETA on this.

Until then, using the mail search to search for your own mail address in the "To:" field may give you a suitable replacement.

**Reply by LMSSonos** [reply]
@ganthern as 1 year passed, i just would like to ask if you have any update on this matter?
